grails-app/assets/javascripts/reporting.js

```diff
@@ -565,7 +565,17 @@ var CategorisedReportsViewModel = function(allReports, order, availableReports,
 var GrantManagerReportsViewModel = function(config) {
     var self = this;
     var projectService = new ProjectService(config.project, config);
-    self.plannedStartDate = ko.observable(config.reportOwner.startDate).extend({simpleDate: false});
+
+    var startDate;
+    var endDate;
+    if (config.project.status == 'Active') {
```

Comment:
Refactor this into projectService.isActive(). (the project service has access to constants for active as well).

Reply:
I am also a bit surprised about the capital "Active" refactoring this into the projectService could encapsulate a case insensitive comparision with the constant.  (The display value is "Active" but the value in the database should be "active").